src/d3a/models/area/__init__.py

```diff
@@ -211,8 +219,8 @@ def area_reconfigure_event(self, **kwargs):
                             export_capacity_kVA=export_capacity_kVA
                         )
 
-        except Exception as e:
-            log.error(str(e))
+        except Exception as ex:
+            log.error(str(ex))
```

Comment:
Just FYI: you can also avoid converting to string. This would be equivalent as far as I know:
```suggestion
        except Exception as ex:
            log.error(ex)
```